torch/_inductor/scheduler.py

```diff
@@ -2134,6 +2134,7 @@ def get_order(n):
     @dynamo_timed
     def codegen(self):
         for node in self.nodes:
+            device = None
```

Comment:
Do you need this anymore and the check below?

Reply:
> Do you need this anymore and the check below?

Yes, some python backend doesn't have device member. It will crash when access device.

Reply:
Oh I lane do you need to have this and the check below anymore, you can just check if you're ready to flush and flush if you are. The default backend will handle it properly

Reply:
1. I need "device" to get backend from "device". and them call the backend's ready_to_flush function.
2. In some backend like "compile_fx", it can't got device, I need check its value is not "None" to avoid crash.

I think current code is make sense. @mlazos